helpers/p2p_docs/doc_data/types/BloomFilter.txt

```diff
@@ -0,0 +1,21 @@
+
+\startformat
+data
+string
```

Comment:
I am not 100% sure that `std::vector<unsigned char>` is string. 
String in my understanding is something human-readable, not just a vector of chars. And this one is interpreted as vector of bytes.

Reply:
From a formal data point of view it shouldn't make a difference, if it's human-readable or just a sequence of bytes, or is there something which would be different there? But it's probably still a good idea to distinguish these to express the intention how it's used. Will replace the string by a vector<unsigned char>.

Reply:
> From a formal data point of view it shouldn't make a difference
From a formal data point `uint32` and `int32` look the same. But it is quite important how you interpret them =)

> or is there something which would be different there?
I guess it is just very subtle and more about people's expectations. If I see a `string` in the protocol - I would expect this to be something I can read in the debugger/message dump.